build.gradle

```diff
@@ -15,17 +15,15 @@
  */
 buildscript {
     ext {
-        dataflowBeamVersion = '2.33.0'
+        dataflowBeamVersion = '2.42.0'
```

Comment:
Latest Bam version is 2.46.0 - https://beam.apache.org/get-started/downloads/

Reply:
Done.